Source/WebCore/platform/graphics/gstreamer/VideoFrameGStreamer.cpp

```diff
@@ -368,6 +368,7 @@ void VideoFrame::copyTo(std::span<uint8_t> destination, VideoPixelFormat pixelFo
     auto* inputCaps = gst_sample_get_caps(sample);
     gst_video_info_from_caps(&inputInfo, inputCaps);
     GstMappedFrame inputFrame(inputBuffer, &inputInfo, GST_MAP_READ);
+    RELEASE_ASSERT(inputFrame);
```

Comment:
Why do you know this can't fail? Shouldn't you handle the !isValid() case instead?

Reply:
@philn should know better, but from what I see this was not being dealt with before so it was mainly not failing because we would see a very weird behavior considering the default answer when it was invalid. That together with tests not failing made me think it was the right choice.

Reply:
It's very unlikely to fail, we should check i think... if the map failed, call `callback({ });` and return early.